tests/tensor_search/test_model_cache_management.py

```diff
@@ -0,0 +1,146 @@
+import torch.cuda
+from tests.marqo_test import MarqoTestCase
+from marqo.s2_inference.s2_inference import _validate_model_properties,\
+    _create_model_cache_key, _update_available_models, available_models, clear_loaded_models
+from marqo.tensor_search.tensor_search import eject_model, get_cuda_info, get_loaded_models
+from marqo.errors import ModelNotInCache, HardwareCompatabilityError
+import psutil
+
+
+
+def load_model(model_name: str, device: str) -> None:
+    validated_model_properties = _validate_model_properties(model_name, None)
+    model_cache_key = _create_model_cache_key(model_name, device, validated_model_properties)
+    _update_available_models(model_cache_key, model_name, validated_model_properties, device, True)
+
+
+class TestModelCacheManagement(MarqoTestCase):
+
```

Comment:
Can we test that a generic model can be loaded, checked (via get...info endpoints) and deloaded, too?

Reply:
added a new test